<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>1100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据可视化</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../OneDrive/图片/Saved Pictures/win11 icons/imageres 187.png</normaloff>../../../OneDrive/图片/Saved Pictures/win11 icons/imageres 187.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox_operate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>1081</height>
    </rect>
   </property>
   <property name="title">
    <string>操作面板</string>
   </property>
   <widget class="QGroupBox" name="groupBox_table_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>表格列表</string>
    </property>
    <widget class="QListWidget" name="list_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>271</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_search_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>当前搜索目录: ./table</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtont_refresh_read">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>290</y>
       <width>110</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_draw">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>291</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>绘图</string>
    </property>
    <widget class="QLabel" name="label_entrys">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>表格条目</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_draw_all">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>全部绘制</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_draw_part">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>绘制选中</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_color">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>300</y>
       <width>61</width>
       <height>39</height>
      </rect>
     </property>
     <property name="text">
      <string>颜色：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_select_color">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>300</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>默认</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>蓝色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>青色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>红色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>品红</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>紫色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>黄色</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>随机颜色</string>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>340</y>
       <width>291</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_type_of_line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>折线图</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_type_of_pot">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>91</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>散点图</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_show_table_name">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>118</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>显示表名</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_show_data_v">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>118</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>显示数据</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_drawing">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>绘制</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_bar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>143</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>柱型图</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cake">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>143</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>饼图</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_subplot_change">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>子图修改</string>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="listWidget_entries">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>967</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton_png">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>143</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>PNG</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_save_img">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>保存图像</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_eps">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>矢量图</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_plot_show">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>1451</width>
     <height>811</height>
    </rect>
   </property>
   <property name="title">
    <string>显示区域</string>
   </property>
   <widget class="QTabWidget" name="tabWidget_tables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1431</width>
      <height>781</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_show">
     <attribute name="title">
      <string>绘图区域</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>1421</width>
        <height>721</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vlayout_show_plot"/>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_table">
     <attribute name="title">
      <string>表格预览</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1431</width>
        <height>741</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>test1</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>test1</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>test</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>830</y>
     <width>1451</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>日志</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1431</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
